ubuntu/14.04/standard/Dockerfile

```diff
@@ -54,6 +54,15 @@ ENV GOROOT_1_9_X64=/usr/local/go1.9.4 \
     GOROOT=/usr/local/go1.10
 ENV PATH $PATH:$GOROOT/bin
 
+# Install HHVM
+RUN apt-get update \
+ && apt-get install software-properties-common apt-transport-https \
```

Comment:
Is the `-y` option needed with this `apt-get install` and the one below, to avoid any prompts?

Reply:
Yes, the -y option prevents prompts. I've added it.